packages/edit-site/src/components/global-styles/screen-block.js

```diff
@@ -6,21 +6,16 @@ import { getBlockType } from '@wordpress/blocks';
 /**
  * Internal dependencies
  */
-import { useGlobalStylesContext } from '../editor/global-styles-provider';
 import ContextMenu from './context-menu';
 import ScreenHeader from './header';
 
 function ScreenBlock( { name } ) {
 	const blockType = getBlockType( name );
-	const { blocks } = useGlobalStylesContext();
 
 	return (
 		<>
 			<ScreenHeader back="/blocks" title={ blockType.title } />
-			<ContextMenu
-				parentMenu={ '/blocks/' + name }
-				context={ blocks[ name ] }
-			/>
+			<ContextMenu parentMenu={ '/blocks/' + name } name={ name } />
```

Comment:
Why is `ContextMenu` named this way? Maybe we can find a name that explains a bit better what's in it?

Reply:
Yes, I struggled to find a good name for this component. (It's the global styles menu for a give context, a context can be "root" or a specific block, potentially other kind of context in the future)

Reply:
Got it! Maybe calling it `GlobalStylesMenu` would be a more clear name? Not big deal anyway.